docs/architecture/incremental-dedupted-story.md

```diff
@@ -0,0 +1,131 @@
+# Incremental - Dedupted Story Sync
+
+## Overview
+
+Airbyte supports syncing data in **Incremental Dedupted Story** mode i.e:
+1. **Incremental** means syncing only replicate _new_ or _modified_ data. This prevents re-fetching data that you have already replicated from a source.
+2. **Dedupted** means that data in the final table will keep only one version of records per primary key (unlike [Append modes](incremental-append.md)). This is determined by sorting the data using the cursor field and keeping only the latest de-duplicated data row. In dimensional data warehouse jargon defined by Ralph Kimball, this is referred as a Slowly Changing Dimension (SCD) table of type 1.
+3. **Story** means that an additional and intermediate table is created where the data is being continuously appended to (with duplicates exactly like [Append modes](incremental-append.md)). With the use of primary key fields, it is identifying effective `start` and `end` dates of each row of a record. In dimensional data warehouse jargon, this is referred as a Slowly Changing Dimension (SCD) table of type 2.
+
+- In this flavor of incremental, records in the warehouse destination will never be deleted in the story tables (named with a `_scd` suffix) but they may get thrown out in the final table.
```

Comment:
is there a reason we are calling them "story" tables, instead of "history" tables? Is this intentional or did this arise because we were playing a game of telephone. @johnlafleur @ChristopheDuong

Reply:
don't block merging based on this, but if it is not intentional let's create a separate issue to fix the name.

Reply:
I think this is also typos introduced at the same time with dedupted... 

Let's switch it to **history** then!